tests/src/Search/Web/ElasticTestCase.php

```diff
@@ -139,6 +139,7 @@ public function getJsonResponse()
     {
         $response = $this->getResponse();
         if (!$response->isOk()) {
+            var_dump($response);
```

Comment:
It is usually preferable to put this into the `fail()` argument? `var_export()` supports returning instead of printing. By the way, isn't there a method for `$response` to give back a readable representation instead of the raw dump?